src/OpenStudy/study_openinterface.jl

```diff
@@ -714,6 +714,7 @@ function _need_update(data::Data, cache)
     end
 end
 
+const _get_dict = Dict{String, Any}()
```

Comment:
ideally `const` should be in capital letters

Reply:
```suggestion
const _GET_DICT = Dict{String, Any}()
```